src/overlays/actors/ovl_En_Time_Tag/z_en_time_tag.c

```diff
@@ -1,7 +1,8 @@
 /*
  * File: z_en_time_tag.c
  * Overlay: ovl_En_Time_Tag
- * Description:
+ * Description: Song of Soaring engraving; Rooftop Oath to Order event; Mikau's and Lulu's diary; Kick out of scene
+ * event
```

Comment:
You can be a bit less abbreviated up here, there's no reason to limit this to a single line. Indeed, several actors have a full description up here.